Assets/Scripts/Camera/CameraManager.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections;
+using System.Collections.Generic;
+using Cinemachine;
+using UnityEngine;
+
+public class CameraManager : MonoBehaviour
+{
+    public static CameraManager cameraManager;
```

Comment:
I don't like doing CameraManager.cameraManager...
```suggestion
    public static CameraManager instance;
```